helix-core/src/test/java/org/apache/helix/manager/zk/client/TestHelixZkClient.java

```diff
@@ -110,33 +110,39 @@ public void testSharingZkClient() throws Exception {
       @Override
       public void handleDataChange(String s, Object o) {
         notificationCountA[0]++;
+        System.out.println("sharedZkClient A increased n0 with path: " + s);
```

Comment:
These outputs are too verbose. Do we really need them?

Reply:
This one is actually worth discussion. Thisone failed rarely maybe twice in hundreds run. Not sure exactly why, and it is really very hard to reproduce, thus, wishing to keep some debug log. 
Let us discuss what we can do with this one.

Reply:
print session id if test fail